source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -580,8 +580,6 @@ def __init__(self):
             parser_post.args[index].location = "json"
 
     @add_debug_info()
-    @clean_links()
```

Comment:
If you wanna be restful, post should create a resource.
Actually, OPTIONS is not in restful defintions...
So...

The good restful way to do it is to create requests resources, and then call it